packages/stryker/src/StrykerCli.ts

```diff
@@ -0,0 +1,93 @@
+import * as program from 'commander';
+import { CONFIG_SYNTAX_HELP } from './ConfigReader';
+import Stryker from './Stryker';
+import StrykerInitializer from './initializer/StrykerInitializer';
+import * as log4js from 'log4js';
+
+const log = log4js.getLogger('stryker-cli');
+let command: string = '', strykerConfig: string | null = null;
+
+export default class StrikerCli {
+
+  private argv: string[];
+
+  constructor(argv: string[]) {
+    this.argv = argv;
```

Comment:
One of the great things about TypeScript is that you don't need this kind of useless code that you see in many languages. You can change the constructor argument to `private argv: string[]` and it will set the private variable `argv` to the argument so you can remove this line, and the declaration of `private argv: string[];` above the constructor 🎉